docs/arlas-collection-model.md

```diff
@@ -61,7 +63,21 @@ The `atom_feed`, `dublin_core_element_name`, `inspire`, `open_search` and `ratse
 | exclude_fields | Comma separated fields names that will be excluded from ARLAS-server responses. By default, none of the fields are excluded | Optional|
 | raster_tile_width | In case the tile is too big, the crop width to apply. Set to -1 if not check must be applied |  Optional |  -1 |
 | raster_tile_height | In case the tile is too big, the crop height to apply. Set to -1 if not check must be applied  |  Optional |  -1 |
-
+| taggable_fields| Comma separated fields names/paths that are allowed to be updated by the [tag service](arlas-api-tagging). By default no field is taggable| Optional|
+
+!!! info "Important 1"
+    Taggable fields paths should not contain `tags`. It's a reserved word.
+    
+!!! info "Important 2"
+    Taggable fields must initially be set to null at indexation in ES.
```

Comment:
Taggable fields must initially be set to a value or to null at index time in ES.